packages/sku/lib/program/options/config/config.option.js

```diff
@@ -0,0 +1,7 @@
+const { Option } = require('commander');
+const configOption = new Option(
+  '-c, --config [config]',
+  'Path to your sku.config.js file',
```

Comment:
Sku config can be written in either a `.js` or `.ts` file, so I try to refer to it agnostic to the file type, i.e. just "sku config".
```suggestion
  'Path to your sku config file',
```